tutorials/tts/FastPitch_Finetuning.ipynb

```diff
@@ -60,7 +60,7 @@
     "BRANCH = 'main'\n",
     "# # If you're using Google Colab and not running locally, uncomment and run this cell.\n",
     "# !apt-get install sox libsndfile1 ffmpeg\n",
-    "# !pip install wget unidecode\n",
+    "# !pip install wget unidecode pynini\n",
```

Comment:
Need to use the pinned version: https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/setup.sh#L5

Reply:
Ah, got it, thanks.